nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-processors/src/main/java/org/apache/nifi/processors/elasticsearch/QueryElasticsearchHttp.java

```diff
@@ -106,6 +121,13 @@
                             + "based on the processor properties and the results of the fetch operation.")
             .build();
 
+    public static final Relationship REL_QUERY_INFO = new Relationship.Builder()
+            .name("query-info")
+            .description(
+                    "Depending on the setting of the Generate Query Info property, a FlowFile is routed to this relationship with " +
```

Comment:
I believe "Generate Query Info" has to be changed to "Routing Strategy for Query Info".

Reply:
nice catch, fixed